src/main/java/org/folio/dao/transactions/BaseTransactionDAO.java

```diff
@@ -88,13 +88,46 @@ public Future<Void> updatePermanentTransactions(List<Transaction> transactions,
     List<JsonObject> jsonTransactions = transactions.stream().map(JsonObject::mapFrom).collect(Collectors.toList());
     String sql = buildUpdatePermanentTransactionQuery(jsonTransactions, conn.getTenantId());
     return conn.execute(sql)
+      .onSuccess(rowSet -> logger.info("updatePermanentTransactions:: success updating permanent transactions"))
+      .onFailure(t -> logger.error("updatePermanentTransactions:: failed updating permanent transactions", t))
```

Comment:
what do you think about adding ids of transactions into log for both success and failure case?

Reply:
We need to add at least the number of transactions to the logs.

Reply:
OK